### DISCUSSION TITLE
Customize custom suggestions flyout container

### DISCUSSION THREAD

**Discussion by zeair** [microsoft-search/pnp-modern-search#2249] [Q&A] [answered]
Hi, Is there a possibility to customize suggestions container? I would like to add some static links on top and bottom, change group name font, display description under display text, change paddings, background color etc....

![image](https://user-images.githubusercontent.com/17876212/180446669-cee960c5-053a-48d1-9f14-7203193b11ef.png)

**Comment by zeair** [comment, marked as answer]
nevermind... I managed :) 
![image](https://user-images.githubusercontent.com/17876212/180559530-aa7381ef-a20f-485f-ac8e-8840edf8d669.png)

**Reply by FranckyC** [reply]
All good then ;)

**Reply by SinghGurd** [reply]
@zeair I assume you are using HTML for displayText property. I had a similar requirement but found that when selected the entire HTML was visible in search box. Did you experience the same behaviour?
